In [32]:
import numpy as np
import pandas as pd
import scipy.stats as stats

In [38]:
data=pd.read_csv("/content/Advertising.csv")
df=pd.DataFrame(data)
df

Unnamed: 0     TV  Radio  Newspaper  Sales
0             1  230.1   37.8       69.2   22.1
1             2   44.5   39.3       45.1   10.4
2             3   17.2   45.9       69.3    9.3
3             4  151.5   41.3       58.5   18.5
4             5  180.8   10.8       58.4   12.9
..          ...    ...    ...        ...    ...
195         196   38.2    3.7       13.8    7.6
196         197   94.2    4.9        8.1    9.7
197         198  177.0    9.3        6.4   12.8
198         199  283.6   42.0       66.2   25.5
199         200  232.1    8.6        8.7   13.4

[200 rows x 5 columns]

In [37]:
#To find the summary statistics in Python we need to use the . describe() method under Pandas before that drop the column Unnamed: 0 
df1=df.drop(["Unnamed: 0"],axis=1)
df1.describe()

TV       Radio   Newspaper       Sales
count  200.000000  200.000000  200.000000  200.000000
mean   147.042500   23.264000   30.554000   14.022500
std     85.854236   14.846809   21.778621    5.217457
min      0.700000    0.000000    0.300000    1.600000
25%     74.375000    9.975000   12.750000   10.375000
50%    149.750000   22.900000   25.750000   12.900000
75%    218.825000   36.525000   45.100000   17.400000
max    296.400000   49.600000  114.000000   27.000000

max = Tv,296.4
min = Radio,0.0
Mean = 147.0425 ,Tv has the highest mean
median = 149.75 ,median highest for TV


The manager wants to find out whether the same amount was spent 
for the three advertisements (TV , Radio and Newspaper).

In [39]:
#H0=Same amount was spent for the three advertisements.
#Ha= Same amount was not spent for the three advertisements. 
#alpha=0.05
f_val,p_val=stats.f_oneway(df1["TV"],df1["Radio"],df1["Newspaper"])
print("f-value =",f_val,"p-value =",p_val)

f-value = 358.8514595342597 p-value = 4.552931539744962e-103


The smaller the p-value, the stronger the evidence that you should reject the null hypothesis. So we can reject the null hypothesis. Therefore same amount was not spent for the three advertisements.